### DISCUSSION TITLE
Rounded corners for Windows

### DISCUSSION THREAD

**Discussion by ArminEskandarii** [flet-dev/flet#1448] [Q&A]
Hi. I have a feature which I think would be really nice. I want that the form corners on Windows be rounded just like Windows 11 but on all versions Windows.

**Comment by FeodorFitsner** [comment]
Then you could make your app's [window frameless](https://flet.dev/docs/controls/page#window_frameless) and draw your own border.

**Reply by tcbuger** [reply]
i just start using flet, and i have a same question. i hide the original window title(border), but i fount that the Page has no border radius property. is there any way to solve this problem?

**Reply by ndonkoHenri** [reply]
Add a `Container` to the page. It has a [`border_radius`](https://flet.dev/docs/controls/container#border_radius) prop.

**Comment by tcbuger** [comment]
> Add a `Container` to the page. It has a [`border_radius`](https://flet.dev/docs/controls/container#border_radius) prop.

yeah  i know that, i saw the WindowDragArea part in project _exampls_, but the relation of _Container_ and _Page_ is " container in the page" right? ... such as this:
![image](https://github.com/flet-dev/flet/assets/52766219/dc153563-427b-41d7-8ec9-90db84edeedc)

**Reply by ndonkoHenri** [reply]
Is it (the image above) what you expected to have? Or you still have some worries...

**Reply by tcbuger** [reply]
of course not... the window is still a right angle window. Notice the top left corner of the window, there's a big blue area. That's the question. And what i want is just like this:
![image](https://github.com/flet-dev/flet/assets/52766219/68899180-107e-4d33-8346-fbba335dde6c)
if you have any solutions to solve this, pls tell me, tkx

**Comment by ArminEskandarii** [comment]
That'll be so great if you could just add something like page_window rounded corners. something like that. And if we set it to True the app should have rounded corners
